### PR TITLE
remove /x32 and /x64 flavors and use instruction scope

### PR DESCRIPTION
**Pull request by williballenthin**
https://github.com/mandiant/capa/pull/961/

### CODE REVIEW

**Review by mr-tz — APPROVED** [review]
Fantastic, thanks for going through all of these!!!

**Review by mike-hunhoff — APPROVED** [review]
great work, thank you @williballenthin !

File: anti-analysis/anti-debugging/debugger-detection/check-for-software-breakpoints.yml [hunk 1]
```diff
@@ -12,12 +12,7 @@ rule:
...
   features:
     - and:
-      - basic block:
-        - and:
-          - mnemonic: cmp
-          - or:
-            - number: 0xCC
-            - and:
-              - number: 0xCD
-              - number: 0x3
```
[the remark this answers is not in the document]
Comment (williballenthin) [reply anti-analysis/anti-debugging/debugger-detection/check-for-software-breakpoints.yml:22]: i can't figure out where they're from. they're not referenced in the associated source code.

i think we need to have documentation for items like these magic constants.

[the remark this answers is not in the document]
Comment (mr-tz) [reply anti-analysis/anti-debugging/debugger-detection/check-for-software-breakpoints.yml:22]: not sure if this is in PMA, but would check for the long form INT 3, see https://anti-debug.checkpoint.com/techniques/assembly.html

[the remark this answers is not in the document]
Comment (mike-hunhoff) [reply anti-analysis/anti-debugging/debugger-detection/check-for-software-breakpoints.yml:22]: @mr-tz correct that was the intent; I should have documented this better 😦

**Comment by williballenthin** [comment]
image

**Pull request merged**